tensorflow/core/util/mkl_util.h

```diff
@@ -1077,6 +1404,21 @@ class MklDnnData {
   void SetIs3DData(bool bIs3D_) { bIs3D = bIs3D_; }
   bool GetIs3D() { return bIs3D; }
 
+#ifdef ENABLE_MKLDNN_V1
+  /// Set user memory primitive using specified dimensions, memory format tag
+  /// and data_buffer. Function automatically uses element data type by using
+  /// input type T used for creating call object.
+  ///
+  /// In a nutshell, function allows user to describe the input tensor to
+  /// an operation. E.g., filter of Conv2D is of shape {1, 2, 3, 4}, and
+  /// memory format tag HWIO, and the buffer that contains actual values is
+  /// pointed by data_buffer.
+  inline void SetUsrMem(const memory::dims& dim, memory::format_tag fm,
```

Comment:
The only difference between this and the v0.x version is that `memory::format` becomes `memory::format_tag`. This is easily handled with a single `#define` at the beginning of the file. Same with  `SetUsrMem` that takes `Tensor*`.

Reply:
Done